spec/features/users_cannot_navigate_to_certain_paths_spec.rb

```diff
@@ -30,7 +31,8 @@
     end
 
     it 'shows 404 error when visiting /admin paths' do
-      visit admin_dashboard_path
+      admin = create(:admin)
+      visit admin_dashboard_path(admin)
```

Comment:
If the admin_dashboard_path is for a user#show page, should the argument here be a user instead of an admin?

Reply:
Yes, did not read the tests carefully enough, good catch